packages/app/src/runner/SpecRunnerHeader.cy.tsx

```diff
@@ -1,18 +1,19 @@
 import SpecRunnerHeader from './SpecRunnerHeader.vue'
-import { useAutStore, useSpecStore } from '../store'
+import { useAutStore } from '../store'
 import { SpecRunnerHeaderFragment, SpecRunnerHeaderFragmentDoc } from '../generated/graphql-test'
 import { createEventManager, createTestAutIframe } from '../../cypress/component/support/ctSupport'
 
 function renderWithGql (gqlVal: SpecRunnerHeaderFragment) {
   const eventManager = createEventManager()
   const autIframe = createTestAutIframe()
 
-  return (<SpecRunnerHeader gql={gqlVal}
+  return (<SpecRunnerHeader
+    gql={gqlVal}
     eventManager={eventManager}
-    getAutIframe={() => autIframe} />)
+    getAutIframe={() => autIframe}/>)
 }
 
-describe('SpecRunnerHeader', () => {
+describe('SpecRunnerHeader', { viewportHeight: 500 }, () => {
```

Comment:
just curious , why is the heigh adjustment needed if the default is 500px already?

Reply:
Our Cypress config for this project sets this values for component testing:

```
viewportWidth: 800,
viewportHeight: 850,
```